renovate.json

```diff
@@ -0,0 +1,20 @@
+{
+    "extends": [
+      "config:base",
+      ":automergeMinor",
+      ":automergeRequireAllStatusChecks",
+      "schedule:earlyMondays"
+    ],
+    "commitMessagePrefix": "renovate - #trivial -",
```

Comment:
Currently I'm unsure if there's an easy way to get the PR title to include the fozzie-components version number, so given we're currently only updating minor deps, I added the #trivial tag to turn off danger checks.

Reply:
Worth noting this is the same config we use in - https://github.com/justeat/eslint-config-fozzie/blob/master/renovate.json